Assets/Scripts/Features/Gameplay/Take Photo/CreatePhotoDoubleTap.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
 /// <summary>
 /// A script to tell the game to take a picture when double tapping.
 /// </summary>
 public class CreatePhotoDoubleTap : MonoBehaviour
 {
 
-    void Update()
+    /// <summary>
+    /// Call once every frame update.
+    /// </summary>
+    private void Update()
```

Comment:
Good that you use private functions, but do not create an XML comment when using a private function/variable

Reply:
Sorry, this is my fault. This is not a problem